mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -1785,6 +1785,45 @@ func TestAccComputeInstanceConfidentialInstanceConfigMain(t *testing.T) {
 	})
 }
 
+<% unless version = 'ga' -%>
+func TestAccComputeInstanceConfidentialHyperDiskBootDisk(t *testing.T) {
+	t.Parallel()
+
+	context_1 := map[string]interface{}{
+		"name":        fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10)),
+		"confidential_compute": true,
+		"kms_key" : acctest.BootstrapKMSKey(t)
+	}
+
+	context_2 := map[string]interface{}{
+		"name":        context_1["random_suffix"],
+		"confidential_compute": false,
+		"kms_key" : acctest.BootstrapKMSKey(t)
+	}
+
+	var instance compute.Instance
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeInstanceConfidentialHyperDiskBootDisk(context_1),
+				Check: resource.ComposeTestCheckFunc(testAccCheckComputeInstanceExists(t, "google_compute_instance.foobar", &instance)
```

Comment:
We can replace `Check`s with `computeInstanceImportStep`, `TestAccComputeInstance_enableDisplay` is an example. This is an old file and doesn't match current practices well, sorry for the churn!